### PR TITLE
JSON importer

### PR DESCRIPTION
**Pull request by izaid**
This PR provides a new example webpage, test_commands.html, which allows users to select an exported JSON file with GLIR commands and it displays them in the browser.

### CODE REVIEW

**Comment by izaid** [comment]
@rossant It is possible that some of the canvas gets cutoff. For instance, I find this with ImageVisual. I think it is to do with the layout in test_commands.html, rather then anything deeper.

**Comment by izaid** [comment]
Also, is it worth including a sample JSON exported from a VisPy demo? If yes, which one?

**Comment by rossant** [comment]
yeah, does display_points work?

**Comment by izaid** [comment]
It does. I put it in the main directory, tell me if you want it elsewhere.

**Comment by rossant** [comment]
you can put it in a new `examples/` folder

**Comment by izaid** [comment]
Okay, done.

File: test_commands.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    cursor: default;
+}
+</style>
+</head>
+
+<body>
+<input type='file' id='browse_button'>
+<canvas id="mycanvas"></canvas>
+</body>
+</html>
```
Comment (rossant) [inline test_commands.html]: add empty line at end of file

File: test_commands.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    r.readAsText(this.files[0]);
+  });
+});
+</script>
+
+<style>
+#mycanvas {
+    width: 100%;
+    height: 100%;
```
Comment (rossant) [inline test_commands.html:44]: can you use a fixed size, e.g. 800x600?

Comment (izaid) [inline test_commands.html:44]: When I use a fixed size, I find that the "Browse" button gets placed below the canvas, even when the canvas is empty. Do you know how to fix this? The "Browse" button should stay at the top.

Comment (rossant) [inline test_commands.html:44]: Hmm. OK forget it for now, we can improve this later. There are a few resize problems to fix anyway (cc @cgestes).

**Comment by rossant** [comment]
LGTM

**Comment by izaid** [comment]
Added the newline. Should we merge this?

@cgestes, if you can help me another time with some of the minor HTML stuff, that would be appreciated.

**Comment by cgestes** [comment]
@izaid what help do you need?

**Pull request merged**